pyshield/radiation/rad_astro.py

```diff
@@ -0,0 +1,546 @@
+import datetime
+import re
+from pathlib import Path
+
+import numpy as np
+
+import ndsl.constants as constants
+from ndsl.dsl.gt4py import FORWARD, acos, computation, cos, interval, sin
+from ndsl.dsl.typing import BoolFieldIJ, Float, FloatFieldIJ, Int
+from ndsl.logging import ndsl_log
+
+
+CCR = 1.3e-6  # iteration limit
```

Comment:
Those will be F64 always. To make them precision dependent

```suggestion
CCR = Float(1.3e-6)  # iteration limit
```